services/management/rds.go

```diff
@@ -367,6 +365,82 @@ func (s *RDSService) AddRDS(ctx context.Context, req *managementpb.AddRDSRequest
 				res.QanMysqlPerfschema = invQANAgent.(*inventorypb.QANMySQLPerfSchemaAgent)
 			}
 
+			return nil
+		// PostgreSQL RDS
+		case managementpb.DiscoverRDSEngine_DISCOVER_RDS_POSTGRESQL:
+			// add PostgreSQL Service
+			service, err := models.AddNewService(tx.Querier, models.PostgreSQLServiceType, &models.AddDBMSServiceParams{
+				ServiceName:    req.ServiceName,
+				NodeID:         node.NodeID,
+				Environment:    req.Environment,
+				Cluster:        req.Cluster,
+				ReplicationSet: req.ReplicationSet,
+				CustomLabels:   req.CustomLabels,
+				Address:        &req.Address,
+				Port:           pointer.ToUint16(uint16(req.Port)),
+			})
+			if err != nil {
+				return err
+			}
+			invService, err := services.ToAPIService(service)
+			if err != nil {
+				return err
+			}
+			res.Postgresql = invService.(*inventorypb.PostgreSQLService)
+
+			_, err = supportedMetricsMode(tx.Querier, managementpb.MetricsMode_PULL, models.PMMServerAgentID)
```

Comment:
let's revert my last comment related to this line.

Reply:
which comment?